eZ/Bundle/EzPublishCoreBundle/Command/ReindexCommand.php

```diff
@@ -387,6 +389,10 @@ private function fetchIteration(Statement $stmt, $iterationCount)
      */
     private function getPhpProcess(array $contentIds, $commit)
     {
+        if (empty($contentIds)) {
+            throw new RuntimeException("'--content-ids=' can not be empty on parallel sub process");
```

Comment:
`InvalidArgumentException` seems to be a better choice here.

Reply:
@adamwojs I think that `RuntimeException` is fine, as at this point `$contentIds` is not an argument, but return value of `fetchIteration` method.

Reply:
@adamwojs `InvalidArgumentException` should be used when the argument is not of an expected type, but here we expect `array` and an `array` it is, but empty. I think that `RuntimeException` is ok.

Reply:
I think @adamwojs is right here, both exception about `--iteration-count` and this one should rather use `InvalidArgumentException` or even better `InvalidArgumentValue` exception.

ok @mateuszbieniek and @adamwojs ?

Reply:
Updated